### ISSUE TITLE
How to populate the data for project?

### ISSUE THREAD

**Issue by wpoosanguansit** [nomicflux/servant-opaleye-blog#7]
Hi,

I am able to run the project but I am getting the data error:

 stack exec blog-tutorial-exe
blog-tutorial-exe: libpq: failed (FATAL:  role "blogtutorial" does not exist
)
 
How should I populate the data along with the schema provided?  Thanks for your help.

**Comment by wpoosanguansit** [comment]
I figured this out.  We just need to create the role and database named blogtutorial in order for the app to exe.

**Comment by nomicflux** [comment]
Thanks for pointing that out - I'll make sure to mention it in the tutorial.

**Comment by nomicflux** [comment]
Lesson 2 now has instructions for creating the user and db.

**Comment by wpoosanguansit** [comment]
Thank you.

**Comment by wpoosanguansit** [comment]
I started fresh and installed Postgresql following the instruction in the doc.  When I do

stack exec blog-tutorial-exe
 
and I posted with the body 

{ username: "username", password: "password" }

access the site at localhost:8080/users I do still get for both post and get:

SqlError {sqlState = "42501", sqlExecStatus = FatalError, sqlErrorMsg = "permission denied for relation users", sqlErrorDetail = "", sqlErrorHint = ""}
 
Is there anything else that would be needed to be setup?  Thanks for your help.
